### PR TITLE
Document pinning commit for Terraform modules

### PR DESCRIPTION
**Pull request by AndiDog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +185,9 @@ Modules in this repository can be used independently:
 ```hcl
 module "kubernetes" {
   source = "github.com/hobby-kube/provisioning/service/kubernetes"
+
+  # Or to pin a certain commit
+  # source = "git::https://github.com/hobby-kube/provisioning.git//service/kubernetes?ref=800d5d5031245cf31a803a147eaa40a0de0573f1"
```
Comment (pstadler) [inline README.md:190]: double slash after .git is probably wrong here

Comment (AndiDog) [reply README.md:190]: That is specifically needed to use the subdirectory of a repo, see [Terraform docs: Modules in Package Sub-directories](https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories).

Comment (pstadler) [reply README.md:190]: my fault. thanks for clarifying.

**Comment by pstadler** [comment]
thank you 🍻

**Pull request merged**